iatidatacube/import_data.py

```diff
@@ -166,24 +169,55 @@ def row_from_csv(row, codelists, reporting_organisation,
 
 
 @timeit
-def import_all_activities(start_at='', end_at='', force_update=False):
+def import_activities_from_csvs(start_at_filename='', end_at_filename='', force_update=False):
+    """Iterates over activity CSV files and imports them into DB using ``import_activities_from_single_csv``
+
+    Iterates over the CSV files produced by ``iatikit`` and stored in ``output/csv/activities``.
+
+    :param start_at_filename: The filename in output/csv to start processing at. If empty, start at the first file.
+    :type start_at_filename: str
+    :param end_at_filename: The filename in output/csv to end processing at. If empty, run until the last file.
+    :type end_at_filename: str
+    :param force_update: If true, activities already in the DB will be deleted and re-imported
```

Comment:
If true, activities already in the DB will be updated, regardless of whether the `hash` column on the activity row has changed or not. This could be useful, for example, if you have changed the import logic for activities.

Reply:
Added a note to that effect, as that's a useful point